spec/models/piece_spec.rb

```diff
@@ -94,4 +92,43 @@
       end
     end
   end
+
+  describe '#move_to!' do
+    let(:piece) { build(:piece) }
```

Comment:
noticing this won't work for the last tests (our piece). Could create new `lets` for that context.